modules/speech-to-text-impl/src/main/java/org/opencastproject/speechtotext/impl/SpeechToTextServiceImpl.java

```diff
@@ -139,6 +140,8 @@ protected String process(Job job) throws Exception {
     List<String> arguments = job.getArguments();
     String language = arguments.get(1);
     URI mediaFile = new URI(arguments.get(0));
+    Boolean translate = Boolean.parseBoolean(arguments.get(2));
```

Comment:
`Boolean.parseBoolean()` does not throw an exception if you enter something other than "true", e.g. "yes" would result in `false`. Please use a parsing method that throws an exception for anything other than valid input strings (so for "true", "false" and the empty string I guess?).